### DISCUSSION TITLE
cpg0034-arevalo-su-motive

### DISCUSSION THREAD

**Discussion by shntnu** [broadinstitute/cellpainting-gallery#86] [Dataset descriptions]
@johnarevalo @eysu35 Let's use this thread to discuss contributing the MOTIVE dataset to cpg.

Please read the [contribution](https://broadinstitute.github.io/cellpainting-gallery/contributing_to_cpg.html) guide to get started. I realize that your data will not include images, only profiles + metadata.

Can you describe what data will be contributed and your proposed structure, to most closely match what we have in there? See https://github.com/broadinstitute/cellpainting-gallery/discussions/62#discussioncomment-6951983 for a related example.

**Comment by johnarevalo** [comment]
We are drafting the docs for the project in https://github.com/broadinstitute/GraphDTI/tree/doc . This is the WIP [dir structure](https://github.com/broadinstitute/GraphDTI/pull/119#issuecomment-2155235541).

**Reply by shntnu** [reply]
Sounds good to me

I'd recommend nesting it similar to this to comply with cpg guidelines:

https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0021-periscope/broad/workspace/publication_data/2022_PERISCOPE/

That is you'd nest it under `s3://cellpainting-gallery/cpg0034-arevalo-su-motive/broad/workspace/publication_data/2024_MOTIVE/`

@ErinWeisbart - the tl;dr is that 100% of the data that @johnarevalo and @eysu35 are going to be transferring is publication data, so to comply with our structure, we'd nest that as `cpgXXXX-<identifier>/<source>/workspace/publication_data/<manuscript_identifier/`. Perhaps we can all chat right after group meeting at noon, Monday, and finalize?

**Reply by ErinWeisbart** [reply]
Summary of our chat: for right now John and Ellen will transfer to the structure Shantunu listed above for `publication_data` as it has a quick deadline for being made public online.
As this is more than just large file storage for a publication and instead represents carefully curated metadata that forms a resource itself, we will think further about moving it to one of the `cpg0016` workspace areas when we have more time.

**Comment by johnarevalo** [comment]
Thanks for the summary, Erin!

@leoank I think we are good to push our data to the staging bucket.
